stage1/prepare-app/prepare-app.c

```diff
@@ -253,6 +256,26 @@ int main(int argc, char *argv[])
 				"Mounting \"%s\" on \"%s\" failed", mnt->source, to);
 	}
 
+	/* Bind mount files, if the source exists */
+	for (i = 0; i < nelems(files_mount_table); i++) {
+		const mount_point *mnt = &files_mount_table[i];
+		int fd;
+
+		exit_if(snprintf(to, sizeof(to), "%s/%s", root, mnt->target) >= sizeof(to),
+			"Path too long: \"%s\"", to);
+		if (access(mnt->source, F_OK) != 0)
+			continue;
+		if (access(to, F_OK) != 0) {
+			pexit_if((fd = creat(to, 0644)) == -1,
+				"Cannot create file: \"%s\"", to);
```

Comment:
Can we get more information in case of an error? Is there an implicit ERRNO parser here?